src/Data/Managers/GetInstanceResources/QueryManager.ts

```diff
@@ -159,6 +159,7 @@ export function InstanceResourcesQueryManager(
         kind: "GetServiceInstances",
         name: serviceEntity,
         pageSize: PageSize.initial,
+        currentPage: { kind: "CurrentPage", value: [] },
```

Comment:
why are you not using the InitialCurrentPage here?

Reply:
Had to miss those, probably to wrong search formula, as I added initialCurrentPage after all those changes